odoo/tools/safe_eval.py

```diff
@@ -104,6 +104,8 @@
     'CALL_METHOD', 'LOAD_METHOD',
     'GET_ITER', 'FOR_ITER', 'YIELD_VALUE',
     'JUMP_FORWARD', 'JUMP_IF_TRUE', 'JUMP_IF_FALSE', 'JUMP_ABSOLUTE',
+    #Added in P3.8
```

Comment:
linting + some extra info from #41085

```suggestion
    # Added in P3.8: https://bugs.python.org/issue17611
```